data/countries/de/bw/data.yaml

```
@@ -175,7 +175,7 @@ state: Baden-Württemberg
 type: REGIONAL_CHAPTER
 urls:
   - type: WEBSITE
-    url: http://www.gruene-hardt.de/
+    url: http://www.gruene-kurpfalz-hardt.de/
```

```suggestion
    url: https://www.gruene-kurpfalz-hardt.de/
```

Fortunately they do support HTTPS.